docs/metrics.md

```diff
@@ -315,6 +315,7 @@ conductor - true for conductor backed actions
 memory - maximum memory allowed for action container
 causedBy - contains the "causedBy" annotation (can be "sequence" or nothing at the moment)
 size - size (in bytes) of the invocation response
+actionStatusCode - status code represents `statusCode` set in result response. (if not set, this field will not be present)
```

Comment:
Should we rename the field into `httpCode` or `httpStatusCode` and point the [statusCode](https://github.com/apache/openwhisk/blob/master/docs/webactions.md#handling-http-requests-with-actions) to the documentation on how the user can set this up?